### PR TITLE
Add tooltips on like buttons

### PR DESCRIPTION
**Pull request by wkhessairi**

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
It seems that the tooltip is missing on the document preview.
Also, please add the french translation.

**Comment by thomasdelhomenie** [comment]
Works fine, except that when you click on the icon, the tooltip is well changed but the style is lost (on document preview, or forum activities for example)

File: extension/war/src/main/webapp/groovy/social/webui/activity/UIDefaultActivity.gtmpl [hunk 1]
```diff
@@ -218,15 +218,17 @@
 								<i class="uiIconComment uiIconLightGray"></i>&nbsp;$allCommentSize
 							</a> 
 						</li>
-					  <% if (uicomponent.isLiked()) { %>
+					  <% if (uicomponent.isLiked()) { 
+					    String unlikeActivityTip = _ctx.appRes("UIActivity.msg.UnlikeActivity"); %>
```
Comment (trongtt) [inline extension/war/src/main/webapp/groovy/social/webui/activity/UIDefaultActivity.gtmpl]: I don't see the need of declare a specific variable `unlikeActivityTip` as it is only used once.

Same for the change in other template files in this PR.

**Comment by thomasdelhomenie** [comment]
One last thing : when I switch to French, the text "Je n'aime plus" is displayed on 2 lines. It should not. The size of the tooltip should be automatically adapted to the text.

**Comment by plamarque** [comment]
is this going to work on document preview in comments panel ?

**Comment by thomasdelhomenie** [comment]
@plamarque yep

**Comment by wkhessairi** [comment]
@thomasdelhomenie OK working on it now

**Pull request closed without merging**